chemprop/cli/train.py

```diff
@@ -110,6 +115,15 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
     # )
 
     transfer_args = parser.add_argument_group("transfer learning args")
+    transfer_args.add_argument(
+        "--checkpoint",
```

Comment:
Will we eventually add the option to restart training from a .ckpt file? Right now the models specified here are loaded using `load_from_file` so the training information (gradients, etc.) are not loaded. So this argument isn't for restarting training from a checkpoint. Maybe later we will add another argument like resume_training` that will require all the paths in `args.checkpoint` to be .ckpt files and will use `load_from_checkpoint` instead.

Reply:
Yes, that is the plan. We left the addition of the `--resume-experiment` flag for v2.1. We can consider assigning someone to work on it.